gravitee-apim-console-webui/src/management/application/details/notification/application-notification.module.ts

```diff
@@ -14,30 +14,20 @@
  * limitations under the License.
  */
 import { NgModule } from '@angular/core';
+import { MatCardModule } from '@angular/material/card';
 import { CommonModule } from '@angular/common';
-import { MatDialogModule } from '@angular/material/dialog';
 import { MatButtonModule } from '@angular/material/button';
-import { MatFormFieldModule } from '@angular/material/form-field';
-import { MatInputModule } from '@angular/material/input';
-import { MatOptionModule } from '@angular/material/core';
-import { MatSelectModule } from '@angular/material/select';
-import { ReactiveFormsModule } from '@angular/forms';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatIconModule } from '@angular/material/icon';
+import { MatLegacySnackBarModule } from '@angular/material/legacy-snack-bar';
 
-import { NotificationSettingsAddDialogComponent } from './notification-settings-add-dialog.component';
+import { ApplicationNotificationComponent } from './application-notification.component';
 
-@NgModule({
-  declarations: [NotificationSettingsAddDialogComponent],
-  exports: [NotificationSettingsAddDialogComponent],
-  imports: [
-    CommonModule,
-    ReactiveFormsModule,
+import { NotificationModule } from '../../../../components/notification';
 
-    MatDialogModule,
-    MatButtonModule,
-    MatFormFieldModule,
-    MatInputModule,
-    MatOptionModule,
-    MatSelectModule,
-  ],
+@NgModule({
+  declarations: [ApplicationNotificationComponent],
+  exports: [],
+  imports: [CommonModule, MatCardModule, MatButtonModule, MatIconModule, MatDialogModule, NotificationModule, MatLegacySnackBarModule],
```

Comment:
we don't need MatLegacySnackBarModule anymore. i think you can juste remove it here